macao-sdk-mirror/src/commonMain/kotlin/com/macaosoftware/plugin/AuthPlugin.kt

```diff
@@ -166,7 +166,11 @@ data class UserData(
     val  facebookLink: String? ="",
     val linkedIn: String?= "",
     val github: String?= ""
-)
+) {
+    // Empty primary constructor required by Firebase
+    constructor() : this("", "", "", "", "", "","","","","")
```

Comment:
Please use nullable instead of empty strings

Reply:
Yes, I tried to use nulls but the compiler was giving an error then I used the empty string.